pkg/controller/controller.go

```diff
@@ -381,7 +381,7 @@ func (c *Controller) runAction(ctx context.Context, as *crv1alpha1.ActionSet, aI
 	if err != nil {
 		return err
 	}
-	phases, err := kanister.GetPhases(*bp, action.Name, *tp)
+	phases, err := kanister.GetPhases(*bp, action.Name, action.Version, *tp)
```

Comment:
what does this default to?

Reply:
Actually, I might have to add some validation here too. It will currently try parsing an empty string and panics.